inlineplz/linters/eslint.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from inlineplz.parsers.base import ParserBase
+from inlineplz.decorators import linter
+import os.path
```

Comment:
[inline-plz]: `prospector: pylint: standard import "import os.path" should be placed before "from inlineplz.parsers.base import ParserBase" (wrong-import-order)`